pkg/kudoctl/util/resources/install.go

```diff
@@ -0,0 +1,115 @@
+package resources
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+)
+
+// Install installs package CRDs.
+// If skipInstance is set to true, only a package's Operator and OperatorVersion is installed.
+func Install(kc *kudo.Client, resources *packages.PackageCRDs, skipInstance bool, instanceName, namespace string, parameters map[string]string) error {
+	// PRE-INSTALLATION SETUP
+	operatorName := resources.Operator.ObjectMeta.Name
+	clog.V(3).Printf("operator name: %v", operatorName)
+	operatorVersion := resources.OperatorVersion.Spec.Version
+	clog.V(3).Printf("operator version: %v", operatorVersion)
+
+	// make sure that our instance object is up to date with overrides from commandline
+	applyInstanceOverrides(resources.Instance, instanceName, parameters)
+	// this validation cannot be done earlier because we need to do it after applying things from commandline
+	if err := validate(resources, skipInstance); err != nil {
+		return err
+	}
+
+	if err := kc.ValidateServerForOperator(resources.Operator); err != nil {
```

Comment:
could we rename this to something like "ValidateServerVersionsForOperator"? :) I really had to go to the implementation to actually understand what this does

I know you did not introduce it but I just found it in your diff so we should probably improve :)

Reply:
Let's tackle this in a different PR where we move this package and also improve on the function names, as mentioned in the comments above. wdyt?